lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -372,8 +374,12 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
         this.formContentClicked.emit(content);
     }
 
-    processDefinitionSelectionChanged(processDefinition) {
-        this.processDefinitionSelection.emit(processDefinition);
+    processDefinitionSelectionChanged(processDefinition: ProcessDefinitionCloud) {
+        if (processDefinition) {
+            const defaultProcessName = this.processNameCloudPipe.transform(this.name, processDefinition);
+            this.processInstanceName.setValue(defaultProcessName);
```

Comment:
You have to call the markAsDirty and markAsTouched or something similar, otherwise, it won't trigger the detect change